sympy/geometry/hyperbola.py

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[ ]:
+
+
+import sympy
+import random
+import math
+import decimal
+import numpy as np
+import matplotlib.pyplot as plt
+from sympy import Symbol, symbols, plot_implicit
+from sympy import Rational
+from sympy.geometry import Point
+from sympy.core.symbol import _symbol
+
+
+class Hyperbola:
```

Comment:
For example, Parabola inherits from GeomerySet. This probably should do that as well to be able to operated correctly.